File: aifupan_project/fupan-server/replay-generic/src/main/java/com/jiuyu/replay/generic/vo/words/AnchorVideoInfoVo.java
package com.jiuyu.replay.generic.vo.words;

import com.jiuyu.replay.generic.vo.video.VideoSliceVo;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

import java.util.Date;
import java.util.List;

@Data
public class AnchorVideoInfoVo {
    /**
     * ID
     */
    @Schema(description = "ID")
    private Long id;
    /**
     * 视频唯一标识
     */
    @Schema(description = "视频唯一标识")
    private String videoId;
    /**
     * 视频名称
     */
    @Schema(description = "视频名称")
    private String videoName;
    /**
     * 开始录制时间
     */
    @Schema(description = "开始录制时间")
    private Date startTime;
    /**
     * 第几段
     */
    @Schema(description = "第几段")
    private Integer paragraph;
    /**
     * 分段类型 0不分段，1按时长分段，2按大小分段
     */
    @Schema(description = "分段类型 0不分段，1按时长分段，2按大小分段")
    private Integer subsectionType;
    /**
     * 视频时长
     */
    @Schema(description = "视频时长")
    private Long duration;
    /**
     * 视频大小，单位：B
     */
    @Schema(description = "视频大小，单位：B")
    private Long vedioSizie;
    /**
     * 结束录制时间
     */
    @Schema(description = "结束录制时间")
    private Date endTime ;
    /**
     * 归属批次号 批次号是指：在录制一个视频中，如果采用分段录制，那么这几段视频归属于同一批次 目前是采集的直播的房间Id, room_id
     */
    @Schema(description = "归属批次号 批次号是指：在录制一个视频中，如果采用分段录制，那么这几段视频归属于同一批次 目前是采集的直播的房间Id, room_id")
    private String batchNumber;
    /**
     * 视频类型  0 ts,1 flv,2 mp4
     */
    @Schema(description = "视频类型  0 ts,1 flv,2 mp4")
    private Integer videoType;
    /**
     * 清晰度 0 标清 1高清 2超清 3 蓝光
     */
    @Schema(description = "清晰度 0 标清 1高清 2超清 3 蓝光")
    private Integer definition;
    /**
     * 存储路径
     */
    @Schema(description = "存储路径")
    private String  storagePath;
    /**
     * 视频url
     */
    @Schema(description = "视频url")
    private String  liveUrl;
    /**
     * 分享地址
     */
    @Schema(description = "分享地址")
    private String  shareUrl;
    /**
     * 在线播放地址
     */
    @Schema(description ="在线播放地址")
    private String playUrl;
    /**
     * 直播源地址
     */
    @Schema(description = "直播源地址")
    private String  sourceUrl;
    /**
     * 主播表主键
     */
    @Schema(description = "主播表主键")
    private Integer  anchorId;
    /**
     * 是否正在录制 0否 1是
     */
    @Schema(description = "是否正在录制 0否 1是")
    private Integer  isRecording;
    /**
     * 直播源类型 0 m3u8  1  flv
     */
    @Schema(description = "直播源类型 0 m3u8  1  flv")
    private Integer  sourceType;
    /**
     * 直播标题
     */
    @Schema(description = "直播标题")
    private String   liveTitle;
    /**
     * 分析状态 0：未分析 1：分析中 2：分析完成 3分析错误
     */
    @Schema(description = "分析状态 0：未分析 1：分析中 2：分析完成 3分析错误")
    private Integer  analysisStatus;
    /**
     * 分析时间
     */
    @Schema(description = "分析时间")
    private String  analysisTime;
    /**
     * 错误原因
     */
    @Schema(description = "错误原因")
    private String  errorReason;
    /**
     * 主播url表的唯一标识
     */
    @Schema(description = " 主播url表的唯一标识")
    private String  secUid;
    /**
     * 行业id
     */
    @Schema(description ="行业id")
    private Long  tradeId;
    /**
     * 平台类型 0：全平台 1：抖音 2：快手 3：视频号
     */
    @Schema(description ="平台类型 0：全平台 1：抖音 2：快手 3：视频号")
    private String  platformType;
    /**
     * 用户id
     */
    @Schema(description ="用户id")
    private Long userId;
    /**
     * 上传的用户昵称
     */
    @Schema(description = "上传的用户昵称")
    private String userNickName;
    /**
     * 类型 1 代表已经录制未上传分享  2 上传分享
     */
    @Schema(description ="类型 1 代表已经录制未上传分享  2 上传分享")
    private Integer type;
    /**
     * 创建时间
     */
    @Schema(description ="创建时间")
    private Date createDate;
    /**
     * 更新时间
     */
    @Schema(description ="更新时间")
    private Date updateDate;
    /**
     * 视频上传状态 0：未上传 1：已上传
     */
    @Schema(description ="视频上传状态 0：未上传 1：已上传")
    private Integer uploadStatus;
    /**
     * 是否已标注敏感词 0：未标注 1：已标注
     */
    @Schema(description ="是否已标注敏感词 0：未标注 1：已标注")
    private Integer isMark;
    /**
     * 占用云空间的大小，单位：M
     */
    @Schema(description ="占用云空间的大小，单位：M")
    private Integer cloudStore;
    /**
     * 视频删除状态 0：未删除 1：已从复盘列表删除 2：已从复盘列表和云空间删除
     */
    @Schema(description ="视频删除状态 0：未删除 1：已从复盘列表删除 2：已从复盘列表和云空间删除")
    private Integer deleteStatus;
    /**
     * 视频的租户id
     */
    @Schema(description ="视频的租户id")
    private Long tenantId;
    /**
     * 分析使用的资源 0：智能分析时长 1：文案提取
     */
    @Schema(description ="分析使用的资源 0：智能分析时长 1：文案提取")
    private Integer useAnalysisPropertyType;
    /**
     * 视频录制的异常状态 0：未出现异常 7：录制网络异常
     */
    @Schema(description ="视频录制的异常状态 0：未出现异常 7：录制网络异常")
    private Integer  recordErrorStatus;
    /**
     * 本地视频文件删除状态 0：未删除 1：已删除
     */
    @Schema(description ="本地视频文件删除状态 0：未删除 1：已删除")
    private Integer localVideoStatus;
    /**
     * 视频切片类型 0：原视频 1：复盘切片视频 2：短视频切片视频
     */
    @Schema(description ="视频切片类型 0：原视频 1：复盘切片视频 2：短视频切片视频")
    private Integer videoSliceType;
    /**
     * 观看人数
     */
    @Schema(description ="观看人数")
    private Integer viewersNum;
    /**
     * 观看人数
     */
    @Schema(description ="观看人数")
    private Integer observationNum;
    /**
     * 最高在线
     */
    @Schema(description = "最高在线")
    private Integer onlineMaxNum;
    /**
     * 销售额区间范围-起始(单位:元)
     */
    @Schema(description = "销售额区间范围-起始(单位:元)")
    private Integer volumeStart;
    /**
     * 销售额区间范围-结束(单位:元)
     */
    @Schema(description = "销售额区间范围-结束(单位:元)")
    private Integer volumeEnd;
    /**
     * 主播信息
     */
    @Schema(description ="主播信息")
    private AnchorUrlInfoVo anchorInfo;

    @Schema(description = "基础设置")
    private BasicSettingsVo basicSettingsVo;
//    /**
//     * 用户主播信息
//     */
//    @Schema(description ="用户主播信息")
//    private AnchorUrlUserVo anchorUserInfo;
    /**
     * 分析信息
     */
    @Schema(description = "分析信息")
    private VideoAnalysisRecordInfoVo recordInfo;
    /**
     * 是否存在弹幕 0：否 1：是
     */
    @Schema(description = "是否存在弹幕 0：否 1：是")
    private Integer existBarrage;
    /**
     * 是否存在数据看板 0：否 1：是
     */
    @Schema(description = "是否存在数据看板 0：否 1：是")
    private Integer existDataBoard;
    /**
     * 是否存在诊断报告 0：否 1：是
     */
    @Schema(description = "是否存在诊断报告 0：否 1：是")
    private Integer hasDiagnosisReport;
    /**
     * 最新诊断报告文件名称
     */
    @Schema(description = "最新诊断报告文件名称")
    private String  diagnosisOssName;
    /**
     * 是否有数据诊断报告
     */
    @Schema(description = "是否有数据诊断报告")
    private Integer hasDataDiagnosisReport;
    /**
     * 最新数据诊断报告文件名称
     */
    @Schema(description = "最新数据诊断报告文件名称")
    private String dataDiagnosisOssName;

    /**
     * 数据诊断已读状态 0未读 1已读
     */
    @Schema(description = "数据诊断已读状态 0未读 1已读")
    private Integer isDataDiagnosisRead;


    /**
     * 是否有笔记小结
     */
    private Integer notesSummary = 0;

    /**
     * 笔记是否存在
     */
    private Integer existsNotes = 0;

    /**
     * 批注是否存在
     */
    private Integer existsMark = 0;

    /**
     * 切片视频的切片信息
     */
    @Schema(description = "切片视频的切片信息")
    private VideoSliceVo videoSliceInfo;

    /**
     * 切片视频所属原视频信息
     */
    @Schema(description = "切片视频所属原视频信息")
    private AnchorVideoInfoVo parentVideoInfo;

    /**
     * 原视频下的所有切片信息
     */
    @Schema(description = "原视频下的所有切片信息")
    private List<VideoSliceVo> sliceList;

    /**
     * 是否有AI优化目的 0：否 1：是
     */
    @Schema(description = "是否有AI优化目的 0：否 1：是")
    private Integer hasAiOptimizePurpose;

    /**
     * 是否设置了星标 0：否 1：是
     */
    @Schema(description = "是否设置了星标 0：否 1：是")
    private Integer hasStar;

    /**
     * 星标信息
     */
    @Schema(description = "星标信息")
    private SourceStarVo sourceStarInfo;

    /**
     * 云空间备注
     */
    @Schema(description ="云空间备注")
    private String cloudRemarks;
    /**
     * 重命名
     */
    @Schema(description ="重命名")
    private String videoRename;
    /**
     * 云空间重命名
     */
    @Schema(description ="云空间重命名")
    private String cloudRename;

    /**
     * 数据来源 0=正常获取 1=巨量拉取
     */
    @Schema(description = "数据来源 0=正常获取 1=巨量拉取")
    private Integer dataSource;
    /**
     * 视频是否下载 0=未下载 1=已下载
     */
    @Schema(description = "视频是否下载 0=未下载 1=已下载")
    private Integer isDownloaded;
}

File: aifupan_project/ReviewAnalysis/vo/AnchorVideoInfoVo.cs
using ReviewAnalysis.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace ReviewAnalysis.vo
{
    public class AnchorVideoInfoVo
    {
        /// <summary>
        /// 主键
        /// </summary>
        [JsonProperty("id")]
        public int Id { get; set; }

        /// <summary>
        /// 视频唯一标识id
        /// </summary>
        [JsonProperty("videoId")]
        public string VideoId { get; set; }

        /// <summary>
        /// 视频文件名称
        /// </summary>
        [JsonProperty("videoName")]
        public string VideoName { get; set; }

        /// <summary>
        /// 开始录制时间 格式：yyyy-MM-dd hh:mm:ss
        /// </summary>
        [JsonProperty("startTime")]
        public string StartTime { get; set; }


        /// <summary>
        /// 第几段视频（每次直播可以分段录制）
        /// </summary>
        [JsonProperty("paragraph")]
        public int Paragraph { get; set; }


        /// <summary>
        /// 分段类型 0不分段，1按时长分段，2按大小分段
        /// </summary>
        [JsonProperty("subsectionType")]
        public int SubsectionType { get; set; }


        /// <summary>
        /// 每段视频的时长 (单位:秒)
        /// </summary>
        [JsonProperty("duration")]
        public string Duration { get; set; }

        /// <summary>
        /// 视频大小  (单位：M)
        /// </summary>
        [JsonProperty("vedioSizie")]
        public string VedioSizie { get; set; }

        /// <summary>
        /// 结束录制时间 end_time 格式：yyyy-MM-dd hh:mm:ss
        /// </summary>
        [JsonProperty("endTime")]
        public string EndTime { get; set; }


        /// <summary>
        /// 归属批次号 批次号是指：在录制一个视频中，如果采用分段录制，那么这几段视频归属于同一批次 目前是直播间的id roomId
        /// </summary>
        [JsonProperty("batchNumber")]
        public string BatchNumber { get; set; }

        /// <summary>
        ///视频类型  0 ts,1 flv,2 mp4
        /// </summary>
        [JsonProperty("videoType")]
        public int VideoType { get; set; }

        /// <summary>
        /// 清晰度 definition 0 标清 1高清 2超清 3 蓝光
        /// </summary>
        [JsonProperty("definition")]
        public int Definition { get; set; }

        /// <summary>
        /// 存储路径 
        /// </summary>
        [JsonProperty("storagePath")]
        public string StoragePath { get; set; }

        /// <summary>
        /// 直播源类型 source_type 0 m3u8 1 flv
        /// </summary>
        [JsonProperty("sourceType")]
        public int SourceType { get; set; }

        /// <summary>
        /// 直播源地址 
        /// </summary>
        [JsonProperty("sourceUrl")]
        public string SourceUrl { get; set; }

        /// <summary>
        /// 主播表主键 
        /// </summary>
        [JsonProperty("anchorId")]
        public int AnchorId { get; set; }

        /// <summary>
        /// 直播标题 live_title
        /// </summary>
        [JsonProperty("liveTitle")]
        public string LiveTitle { get; set; }

        /// <summary>
        /// 是否正在录制 is_recording 0否 1是
        /// </summary>
        [JsonProperty("isRecording")]
        public int IsRecording { get; set; }

        /// <summary>
        /// 分析完成时间 analysis_time 格式：yyyy-MM-dd hh:mm:ss
        /// </summary>
        [JsonProperty("analysisTime")]
        public string AnalysisTime { get; set; }

        /// <summary>
        /// 分析状态 analysis_status 0：未分析 1：分析中 2：分析完成 3：分析失败
        /// </summary>
        [JsonProperty("analysisStatus")]
        public int AnalysisStatus { get; set; }
        /// <summary>
        /// 分析失败原因
        /// </summary>
        [JsonProperty("errorReason")]
        public string ErrorReason { get; set; }

        /// <summary>
        /// 主播的唯一Id
        /// </summary>
        [JsonProperty("secUid")]
        public string SecUid { get; set; }

        /// <summary>
        /// 行业id
        /// </summary>
        [JsonProperty("tradeId")]
        public string TradeId { get; set; }
        /// <summary>
        /// 平台类型
        /// </summary>
        [JsonProperty("platformType")]
        public string PlatformType { get; set; }
        /// <summary>
        /// 用户id
        /// </summary>
        [JsonProperty("userId")]
        public string UserId { get; set; }
        /// <summary>
        /// 是否已上传到服务器 0：否 1：是
        /// </summary>
        [JsonProperty("uploadStatus")]
        public int UploadStatus { get; set; }
        /// <summary>
        /// 是否已标注词语 0：否 1：是
        /// </summary>
        [JsonProperty("isMark")]
        public int IsMark { get; set; }
        /// <summary>
        /// 在线复盘url
        /// </summary>
        [JsonProperty("shareUrl")]
        public string ShareUrl { get; set; }
        /// <summary>
        /// 在线文件的url
        /// </summary>
        [JsonProperty("playUrl")]
        public string PlayUrl { get; set; }
        /// <summary>
        /// 占用云空间的大小，单位：M
        /// </summary>
        [JsonProperty("cloudStore")]
        public int CloudStore { get; set; }
        /// <summary>
        /// 视频删除状态 0：未删除 1：已从复盘列表删除 2：已从复盘列表和云空间删除
        /// </summary>
        [JsonProperty("deleteStatus")]
        public int DeleteStatus { get; set; }
        /// <summary>
        /// 租户id
        /// </summary>
        [JsonProperty("tenantId")]
        public long TenantId { get; set; }
        /// <summary>
        /// 主播信息
        /// </summary>
        [JsonProperty("anchorInfo")]
        public AnchorInfo AnchorInfo {  get; set; }
    }
}
